msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -374,54 +307,288 @@ Example for RSC in a chat
+
+## Add RSC permissions to your Teams application
+
+1. [Register your app with Microsoft identity platform using the Azure AD portal](#register-your-app-with-microsoft-identity-platform-using-the-azure-ad-portal).
+1. [Add application and delegated RSC permission to your app manifest](#update-your-teams-app-manifest).
+1. [Review your application permissions in the Azure AD portal](#review-your-application-permissions-in-the-azure-ad-portal).
+
+## Verify application RSC permission granted to your app
+
+### Prerequisite
+
+1. [Configure consent settings](#configure-consent-settings).
+    * Configure group owner consent settings for RSC in a team using the Azure AD portal.
+    * Configure chat owner consent settings for RSC in a chat using the Microsoft Graph APIs.
+
+To verify the application RSC permissions, follow these steps:
+
+1. [Install your app in Teams](#sideload-your-app-in-teams).
+1. [Obtain an access token from the identity platform](#obtain-an-access-token-from-the-microsoft-identity-platform).
+1. [Check the RSC permissions granted to a specific resource](#check-the-rsc-permissions-granted-to-a-specific-resource).
+    * Check your app for added RSC permissions in a team.
+    * Check your app for added RSC permissions in a chat.
+
+## Configure consent settings
+
+For application permissions, admins can enable or disable RSC permissions for all apps for the entire tenant using a toggle in Teams admin center. The tenant-level controls of app-only RSC permissions differ based on resource type.
+
+For delegated permissions, any authorized user can consent to the permissions requested by the app.
 
 <br>
+<details>
+
+<summary><b>Configure group owner consent settings for RSC in a team using the Azure AD portal</b></summary>
+
+You can enable or disable [group owner consent](/azure/active-directory/manage-apps/configure-user-consent-groups?tabs=azure-portal) directly within the Microsoft Azure portal:
+
+1. Sign in to the [Azure portal](https://portal.azure.com) as a [Global Administrator or Company Administrator](/azure/active-directory/roles/permissions-reference#global-administrator&preserve-view=true).
+1. Select **Azure Active Directory** > **Enterprise applications** > **Consent and permissions** > [**User consent settings**](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ConsentPoliciesMenuBlade/UserSettings).
+1. Enable, disable, or limit user consent with the control labeled **Group owner consent for apps accessing data**. The default is **Allow group owner consent for all group owners**. For a team owner to install an app using RSC, group owner consent must be enabled for that user.
+
+    :::image type="content" source="../../assets/images/azure-rsc-team-configuration.png" alt-text="Screenshot shows the Azure RSC team configuration.":::
```

Comment:
Is this link working? I dont see it in the preview.

Reply:
Yes checked and its working